features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APICreateMenu.jsx

```diff
@@ -1,3 +1,21 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Year should be 2020

Reply:
https://github.com/wso2/carbon-apimgt/pull/8279/commits/4c0c10bd001c827254f1e9958dc6ca007f4ef4c3